java-connectors/kafka-connect-azure-eventhubs/src/test/java/io/lenses/java/streamreactor/connect/azure/eventhubs/source/EventHubsKafkaConsumerControllerTest.java

```diff
@@ -16,78 +19,35 @@
-  //private static final String SOME_NAME = "somename";
-  //private AzureEventHubsConfig hubsConfig;
-
-  private ArrayBlockingQueue<ConsumerRecords<String, String>> recordsQueue;
+  private ArrayBlockingQueue<ConsumerRecords<Object, Object>> recordsQueue;
 
   private EventHubsKafkaConsumerController testObj;
 
 
   @BeforeEach
   void setUp() {
-    //hubsConfig = mock(AzureEventHubsConfig.class);
     recordsQueue = new ArrayBlockingQueue<>(10);
   }
 
-  //@Test
-  //void createShouldFetchRelevantPropertiesAndCallForConsumerCreation() { //TODO remove
-  //  //given
-  //  mockPropertiesReturn();
-  //  List<String> bootstrapServers = Collections.singletonList("some.server:9393");
-  //  String groupId = "group.id";
-  //  Password jaas = new Password("jaas.pass");
-  //  String saasMechanism = "saasMechanism";
-  //  String secProtocol = "secProtocol";
-  //  String eventHubName = "eventHubTopic";
-  //  Class stringDeserializer = StringDeserializer.class;
-  //
-  //  //when
-  //  testObj = new EventHubsKafkaConsumerController(hubsConfig, consumerProvider, recordsQueue);
-  //
-  //  //then
-  //  verify(consumerProvider).createProducer(any(AzureEventHubsConfig.class), eq(recordsQueue));
-  //}
-
-  //private void mockPropertiesReturn() {
-  //  List<String> bootstrapServers = Collections.singletonList("some.server:9393");
-  //  String groupId = "group.id";
-  //  String eventHubName = "eventHubTopic";
-  //  Password jaas = new Password("jaas.pass");
-  //  String saasMechanism = "saasMechanism";
-  //  String secProtocol = "secProtocol";
-  //  Class stringDeserializer = StringDeserializer.class;
-  //  when(this.hubsConfig.getList(
-  //      getPrefixedKafkaConsumerConfigKey(BOOTSTRAP_SERVERS_CONFIG))).thenReturn(bootstrapServers);
-  //  when(this.hubsConfig.getString(
-  //      AzureEventHubsConfigConstants.EVENTHUB_NAME)).thenReturn(eventHubName);
-  //  when(this.hubsConfig.getString(
-  //      getPrefixedKafkaConsumerConfigKey(GROUP_ID_CONFIG))).thenReturn(groupId);
-  //  when(this.hubsConfig.getPassword(
-  //      getPrefixedKafkaConsumerConfigKey(SaslConfigs.SASL_JAAS_CONFIG))).thenReturn(jaas);
-  //  when(this.hubsConfig.getString(
-  //      getPrefixedKafkaConsumerConfigKey(SaslConfigs.SASL_MECHANISM))).thenReturn(saasMechanism);
-  //  when(this.hubsConfig.getString(
-  //      getPrefixedKafkaConsumerConfigKey(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG))).thenReturn(secProtocol);
-  //  when(this.hubsConfig.getClass(
-  //      getPrefixedKafkaConsumerConfigKey(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG))).thenReturn(stringDeserializer);
-  //  when(this.hubsConfig.getClass(
-  //      getPrefixedKafkaConsumerConfigKey(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG))).thenReturn(stringDeserializer);
-  //}
-
   @Test
   void pollShouldPollQueueAndReturnSourceRecords() throws InterruptedException {
     //given
     Duration duration = Duration.of(2, ChronoUnit.SECONDS);
```

Comment:
see above comments on test.  Looks like this code is very similar so maybe you could extract some reusable code here??

Reply:
Tidied mocks up a little, thanks!